de-lib/src/main/java/org/iplantc/de/desktop/client/views/windows/AboutApplicationWindow.java

```diff
@@ -48,10 +50,15 @@
 
     @Inject
     AboutApplicationWindow(final AboutApplicationServiceAsync aboutApplicationService,
-                           final AboutApplicationAppearance appearance) {
+                           final AboutApplicationAppearance appearance, final UserInfo userInfo) {
         this.aboutApplicationService = aboutApplicationService;
         this.appearance = appearance;
-        setSize("320", "260");
+        this.userInfo = userInfo;
+
+        String width = getSavedWidth(WindowType.ABOUT.toString());
+        String height = getSavedHeight(WindowType.ABOUT.toString());
+        setSize((width == null) ? "320" : width, (height == null) ? "260" : height);
```

Comment:
It might be safer to check the saved width/height with `Strings.isNullOrEmpty` (in case an empty string gets saved for some reason).
This also might be a good opportunity to move the default values into an appearance.

Reply:
👍